README.md

```diff
@@ -260,138 +260,168 @@ We gladly credit all contributors.
-    some companies do see value in offering (mostly limited) opportunities for sale.
+    [in recent years](https://techcrunch.com/2015/10/14/selling-private-company-shares-2-0/).
+    Some companies do see value in offering (mostly limited) opportunities for sale.
     [SharesPost](http://sharespost.com/), [Equidate](https://www.equidateinc.com/), and
     [EquityZen](https://equityzen.com/) have sought to establish a market around secondary sales,
     particularly for well-known pre-IPO companies.
-    A few other secondary firms have emerged that have interest in certain purchases,
-    especially for larger sales from founders, early employees, or executives (parties
-    generally in possession of larger shares).
-    A company can work with the firm to facilitate multiple transactions.
-    These firms include [137 Ventures](http://137ventures.com/),
-    [ESO Fund](https://employeestockoptions.com/),
-    [Akkadian Ventures](https://www.akkadianventures.com/),
-    [Industry Ventures](http://www.industryventures.com/), [Atlas Peak](http://www.atlaspeakcap.com/),
-    and [Founders Circle](http://www.founderscircle.com/).
+    \- 🚧 Briefly share what these companies see as valuable in offering opportunities for
+    private sale.
+    - A few other secondary firms have emerged that have interest in certain purchases,
+      especially for larger sales from founders, early employees, or executives (parties
+      generally in possession of larger shares).
+      A company can work with the firm to facilitate multiple transactions.
+      These firms include [137 Ventures](http://137ventures.com/),
+      [ESO Fund](https://employeestockoptions.com/),
+      [Akkadian Ventures](https://www.akkadianventures.com/),
+      [Industry Ventures](http://www.industryventures.com/), [Atlas Peak](http://www.atlaspeakcap.com/),
+      and [Founders Circle](http://www.founderscircle.com/).
 
 ### How does the value of equity change?
 
 We’ve now seen how hard it is to evaluate the value of equity in private companies,
 because often stock can’t be sold.
+
+- 🚧 So this is actually not currently in the preceding section.
+  Why and how does whether stock can be sold affect its value at all?
+
 Understanding equity is more complex than knowing its “true value” at a single point in
 time. The intention of giving shareholders equity is to grow the company—it’s about the
 future. So the value of your equity is a reflection of three major concerns:
 
-1. How well the company is doing now—that is, how profitable it is
-2. How well the company will perform in the future
+1. How well the company is doing now—that is, how profitable it is.
+2. How well the company will perform in the future.
 3. How likely it is the company will be valuable as part of another company—that is, whether
-   it may be [acquired](https://www.investopedia.com/terms/a/acquisition.asp)
+   it may be [acquired](https://www.investopedia.com/terms/a/acquisition.asp). - 🚧 Add
```

Comment:
Another TK point not formatted right. In this case it should be its own paragraph.